e2e/tests/init_test.go

```diff
@@ -0,0 +1,70 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/cucumber/godog"
```

Comment:
We should prefer Regen Network's `gocuke` instead of `godog`:

https://github.com/regen-network/gocuke